modules/private-clusters-about-aws.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * installing/
+
+[id="private-clusters-about-aws_{context}"]
+= Private clusters in AWS
+
+To create a private cluster on Amazon Web Services (AWS), you must provide an existing private VPC and subnets to host the cluster. The installation program must also be able to resolve the DNS records that the cluster requires. The installation program configures the Ingress Operator and API server for only internal traffic.
```

Comment:
> only internal traffic

Maybe replace it with something closer to, ` for access from the private network only`